lighthouse-core/test/results/sample_v2.json

```diff
@@ -5060,6 +5060,23 @@
       "warningHeader": "Warnings: "
     },
     "icuMessagePaths": {
```

Comment:
> icuMessagePaths

😱gets a quite a bit longer on each of these PRs :)

Reply:
retweet 🐦

Reply:
We could make ICUmsg paths optional, only add it when requested... 🤷‍♂